frontend/javascripts/oxalis/model_initialization.ts

```diff
@@ -569,6 +586,7 @@ function determineDefaultState(
 
   if (someTracing != null) {
     position = Utils.point3ToVector3(someTracing.editPosition);
+    additionalCoordinates = someTracing.editPositionAdditionalCoordinates;
   }
 
   if (urlStatePosition != null) {
```

Comment:
This should also use the `additionalCoordinates` that might have been part of the url state

Reply:
good catch!